### DISCUSSION TITLE
Datasmith Exporter??

### DISCUSSION THREAD

**Discussion by jayb8200** [iTwin/community#40] [Q&A]
I am looking to take data from Synchro and visualize it in Unreal Engine. I found an exporter called datasmith but every link I find says the page no longer exists. Am I possibly missing something?

**Comment by aruniverse** [comment]
We launched the iTwin Exporter for Datasmith as a Technology Preview; we received great feedback from early adopters. One of the most significant drawbacks of the iTwin Exporter for Datasmith that we received feedback about was that the file format is unsuitable for large-scale infrastructure projects. One of the core tenets of the iTwin Platform is scalable support for large-scale modern infrastructure projects. With this limitation identified, we are developing new technology, which supports large-scale infrastructure model streaming in Unreal Engine.

To get started with the new experimental technology, register an application on the iTwin Platform by following [these steps](https://developer.bentley.com/tutorials/quickstart-web-and-service-apps/#12-register-your-application).

You can get more information about how to load your iTwin in Unreal at our [Unreal plugin's page](https://github.com/iTwin/unreal-engine-3dft-plugin)
